README.md

```diff
@@ -52,6 +52,73 @@ following fields:
 * `events`: An event sink which supports the same interface as Node's `EventEmitter`. Required. See
   below for the events that get generated.
 
+### Emitting metadata about the document
+
+Every time the validator finds/deduces a piece of metadata about the document, it emits a `metadata` event.
+`metadata` messages contain two arguments: *key* and *value*.
+Keys are unique ID's, while the types of values are different according to the specific kind of metadata.
```

Comment:
Cheers! (It wasn't a typo, anyway; I've always written plurals of acronyms that way&nbsp;&mdash;&nbsp;and I thought it was correct! I learnt and reasoned just [like this guy here](http://english.stackexchange.com/questions/55970/plurals-of-acronyms-letters-numbers-use-an-apostrophe-or-not).)